### ISSUE TITLE
Content review request for notification settings from Authenticated Experience

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#22529]
### Content support for:
We're adding a new section to VA.gov profile to allow users to set notification settings.  This request is for content review of:
-  the section title (currently "notification settings")
- content communicating where the notifications will go
- name of section where ungrouped notifications will go (currently "other notifications")
- error messages

All of the copy in the notification groups ("Healthcare notifications" is a group example in the mock-up), and content inside those will come from VA Profile's API.  What is shown in the mock-ups is placeholder copy that doesn't need review at this time.

## How to submit your request

- [x] Complete each area of this intake form that applies to your team's needs. Provide as much detail as you can so we can fully understand your request.
- [ ] Post a link to this issue in [#vsa-public-websites](https://dsva.slack.com/channels/vsa-public-websites) in Slack, tag @RLHecht and @brianalloyd. If your request includes engineering support, also tag @ncksllvn and @kelsonic.
- [ ] If noted in the guidance below, please schedule any relevant meetings.

## About your app/tool/functionality

* **Is this request related to a VA.gov tool? If yes, provide a brief description of the tool or functionality:**  
   This will be a new section in the VA.gov profile - "Notification settings".  
   
* **Provide links to any available collateral: product briefs/outlines, wireframes, prototypes, user research synthesis:** 
   [Prototype](https://preview.uxpin.com/bb87d0fa61a32938a47e7bcdc836db235ab77576#/pages/137989653/simulate/sitemap?mode=i)
---

## About your support needs
*Please select all that apply, and follow related guidance.* 

### Content
- [ ] **Writing support for new app/tool** 
  Since we're adding something to an existing tool, I'm selecting editing support.
  
- [x] **Editing support for app/tool** <br> <br> **Note:** We'll work with your team and the VSP content resource to determine how best to support your needs.
   * URL where draft content appears (if available) <br>    
     - [Prototype](https://preview.uxpin.com/bb87d0fa61a32938a47e7bcdc836db235ab77576#/pages/137989653/simulate/sitemap?mode=i)      
   * **Design context** <br> **Example:** Is this content in a user flow, an in-line error message, tool tip, field label?
     - This content is in a user flow.  There's one primary screen that has a read / edit mode. 
     - User navigates to notification settings section > sees list > clicks edit > makes changes > saves > back to list
     - The mock-up also contains an "autosave" version. We are not planning on this for the MVP but it's a concept we've discussed at length and want to keep in our back pockets as an option, if usability testing outcomes dictate that we need to pivot in this direction. It'd be helpful to get feedback on this option too.
     
   * **Scope: How many screens or messages?** 
     - Primary screen has: h1, intro text 
     - We'll need review of error messages:
        - **When we can't load a users notifications, or a users notifications and personal information**: We’re sorry. Something went wrong on our end and we can’t access your notification settings at this time. Please try again later.
        - **When we can't load a users personal information**: We’re sorry. Something went wrong on our end and we can’t access your contact information at this time. Please try again later.
        -  **When we can't save a users changes**: We’re sorry. We’re having issues connecting to our database and couldn’t update your notifications at this time. Please try again.

### Frontend engineering support 

n/a

### Desired/target implementation date

What's the ETA for the app/tool launch or the date when you need this request completed? Our team's capacity will determine which sprint the work falls into.

4/7/2021

---

### About your team

- Team name: Auth Exp
- DEPO product owner:  Samara Strauss
- Product manager: AJ (Anastasia) Jakabcin
- Designer: Liz Lantz
- FE engineer: Erik Hansen, Sandra Hallie
- Product/team Slack channel: #vsa-authd-exp

--- 
#### Acceptance Criteria
- [ ] Convert this request into a Zenhub epic
- [ ] Follow-up with requestor to address any lingering questions/concerns
- [ ] Create epic/tickets as needed for the Public Websites team to implement

#### Definition of Done
* Request ticket is closed and implementation tickets created as needed

**Comment by brianalloyd** [comment]
@RLHecht this appears to be mainly (if not completely) content support.  Going to add the `Sitewide team` labels in support.  Thanks.

**Comment by andaleliz** [comment]
Heads up, I'm going to be adding a screen w/ a success alert. In case you get to the content review before I have a chance to update the mock-ups, the proposed message will be:

`We have updated your notification settings.`

**Comment by andaleliz** [comment]
I've added screens with error messages but only for the mobile version. Will add to desktop once we have final copy.

**Comment by andaleliz** [comment]
@RLHecht and @DanielleThierryUSDSVA reviewed this with me during our suite office hours. We're good to go on all the content with one proposed exception, to handle the use case of "user is missing contact information".  We talked about displaying it in an info alert. I need to chat w/ the team about a couple of things before I have suggested copy ready. Will come back here w/ options when I've got that.  Thanks!
